### PR TITLE
Update Spotless and Ktlint, and appease these robots

### PR DESCRIPTION
**Pull request by swankjesse**
I think it’s worse in general, but I’d rather have the inelegant formatting of a robot than wasting human attention on whitespace.

### CODE REVIEW

File: okio/src/commonMain/kotlin/okio/internal/Buffer.kt [hunk 1]
```diff
@@ -461,33 +463,61 @@ internal inline fun Buffer.commonWriteDecimalLong(v: Long): Buffer {
...
-    else if (v < 1000000000000000L)
-      if (v < 10000000000000L) 13
-      else if (v < 100000000000000L) 14
-      else 15
-    else if (v < 100000000000000000L)
-      if (v < 10000000000000000L) 16
-      else 17
-    else if (v < 1000000000000000000L) 18
-    else 19
+    if (v < 100000000L) {
```
Comment (swankjesse) [inline okio/src/commonMain/kotlin/okio/internal/Buffer.kt:466]: Yech

File: okio/src/jvmTest/kotlin/okio/BufferFactory.kt [hunk 2]
```diff
@@ -59,7 +59,8 @@ enum class BufferFactory {
 
       return bufferWithRandomSegmentLayout(dice, largeByteArray)
     }
-  };
+  },
+  ;
```
Comment (swankjesse) [inline okio/src/jvmTest/kotlin/okio/BufferFactory.kt:63]: Arguably a ktlint bug

Comment (JakeWharton) [reply okio/src/jvmTest/kotlin/okio/BufferFactory.kt:63]: I think this looks right. Trailing comma means clean diffs. Semicolon is required to separate constants from functions.

Comment (JakeWharton) [reply okio/src/jvmTest/kotlin/okio/BufferFactory.kt:63]: I think this looks right. Trailing comma means clean diffs. Semicolon is required to separate constants from functions.

File: okio/src/commonMain/kotlin/okio/internal/Buffer.kt (hunk 1 above)
Comment (JakeWharton) [reply okio/src/commonMain/kotlin/okio/internal/Buffer.kt:466]: Both versions are pretty bad.

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**